packages/wallets/src/AppContent.scss

```diff
@@ -6,11 +6,12 @@
     width: 100%;
     align-self: stretch;
     background: var(--system-light-7-secondary-background, #f2f3f4);
-    min-height: calc(var(--wallets-vh, 1vh) * 100 - 8.4rem); // 100vh - (4.8rem header + 3.6rem footer)
+    min-height: calc(100vh - 8.4rem); // 100vh - (4.8rem header + 3.6rem footer)
 
     @include mobile {
         overflow-y: auto;
         height: calc(var(--wallets-vh, 1vh) * 100 - 4rem); // 100vh - 4rem (header)
+        min-height: calc(var(--wallets-vh, 1vh) * 100 - 8.4rem); // 100vh - (4.8rem header + 3.6rem footer)
```

Comment:
Have moved the 9th line into the `mobile` mixin as it was intended to be added only in it to fix iOS issues in https://github.com/binary-com/deriv-app/pull/11932